### DISCUSSION TITLE
How to interpret the simulation results

### DISCUSSION THREAD

**Discussion by ryanl54** [asreview/asreview#1252] [Q&A] [answered]
Hello ASReview Team,

I ran two simulations with different prior knowledge sizes: A) 1 relevant + 1 irrelevant; B) 100 relevant + 100 irrelevant. My dataset includes 21417 citations in total with 552 relevant records, and 20865 irrelevant records. Could you please help me with the following two questions?

Question 1:
After I exported the CSV files ("ASReviewLABprogressRecall") of the two simulations' results, it make sense that for the "Relevant by ASReview LAB" column in the CSV files, the max number of simulation A is 551 (552 - 1 relevant prior knowledge), the max number of simulation B is 452 (552 - 100 relevant prior knowledge). I also attached the two files .

However, why are their total rows the same (both files have 21417 records)? Shouldn't the simulation A have only 21415 records (21417 - 2 prior knowledge) while the simulation B have 21217 records (21417 - 200 prior knowledge)? (because simulation needs to exclude the prior knowledge records)

Question 2:
In the "Analytics" section, what does "Irrelevant records since last relevant" mean? For simulation B, this value is 212 (please see the screenshot below). However, in the CSV file of Simulation B (please see the column B in the attached "ASReviewLABprogressRecall_100" file), the max number (452) appears in row 21006, which means there are 413 irrelevant records after it. Shouldn't these two numbers (212 and 413) be the same? What is the correct way to interpret these two results?

Thanks very much!

<img width="677" alt="image" src="https://user-images.githubusercontent.com/116648730/197776649-359991db-493d-4929-a933-05dd296117ce.png">

[ASReviewLABprogressRecall_1.csv](https://github.com/asreview/asreview/files/9860340/ASReviewLABprogressRecall_1.csv)
[ASReviewLABprogressRecall_100.csv](https://github.com/asreview/asreview/files/9860344/ASReviewLABprogressRecall_100.csv)

**Comment by Rensvandeschoot** [comment, marked as answer]
Thank you for sharing the data! It is the data used for constructing the recall plot. This plot has been developed by @terrymyc, who can explain how the information is used for the graph. 

For inspecting the simulation results, I recommend [exporting](https://asreview.readthedocs.io/en/latest/progress.html#export-results) the `.asreview` project file and using the [`insights`](https://github.com/asreview/asreview-insights#asreview-insights) package to create plots, compute metrics and much more. The content of the `.asreview` file, including the order in which the records have been labeled (labeling time) and whether the record is used as prior (see code snippet [7] and the label `-1` in the example output), can be inspected via the [API](https://asreview.readthedocs.io/en/latest/example_api_asreview_file.html).

**Reply by ryanl54** [reply]
Thank you very much!
